setup/README.md

```diff
@@ -17,14 +17,16 @@ With all options set:
   with:
     os: linux
     version: v0.1.14
+    sha256sum: 407875b9e2263fb3ca94a6be166cd280f92404f01b1a00989a6deec441635706
```

Comment:
This usage config uses the local action?

Reply:
@itchyny 
Indeed, that’s strange. I was expecting an action from an external repository, so I’ll fix the usage.

Reply:
This is `setup` action, not `exec` action.